docs/rules/no-static-imports-of-user-specific-scoped-modules-in-ssrable-components.md

```diff
@@ -0,0 +1,62 @@
+# Deprecation Notice for Salesforce Scoped Modules during Server-Side Rendering (SSR)
+
+Static imports of user-specific scoped modules, such as `@salesforce/user/*`, are not supported in LWC components marked with `lightning__ServerRenderable` or `lightning__ServerRenderableWithHydration`.
+
+## Rule details
+
+The following Salesforce scoped modules are deprecated when using Server-Side Rendering (SSR):
+
+-   `@salesforce/user/*`
+-   `@salesforce/userPermission/*`
+-   `@salesforce/customPermission/*`
+-   `@salesforce/featureFlag/*`
+
+To replace these deprecated modules, it is recommended to use a **data provider** to fetch the required information declaratively. Avoid using the dynamic import pattern as shown below, as it is considered an **anti-pattern**.
```

Comment:
There is a suggestion here for a link to use data providers: https://github.com/salesforce/eslint-plugin-lwc/pull/164#discussion_r1795990377

@nrobertdehault Is there public doc for the `User` data binding we can use here?

Reply:
I think we definitely need at least a link to the documentation and/or an example.

Reply:
@tedconn

Reply:
@lpomerleau , are you OK to stick with dynamic imports for now?

Reply:
The word "deprecated" refers to an API that is marked for future removal but still works for now. In this case, the issue seems to be more about the availability of the API in different environments.

Reply:
Here is the documentation for available expressions: https://developer.salesforce.com/docs/atlas.en-us.exp_cloud_lwr.meta/exp_cloud_lwr/advanced_expressions.htm

Reply:
> are you OK to stick with dynamic imports for now?

Yes. It's the only reasonable workaround